src/framework/ui/internal/uiengine.h

```diff
@@ -52,8 +53,10 @@ class UiEngine : public QObject, public IUiEngine, public Injectable
 
     // for internal use
     Q_PROPERTY(InteractiveProvider * _interactiveProvider READ interactiveProvider_property CONSTANT)
+    Q_PROPERTY(muse::tours::ToursProvider * _toursProvider READ toursProvider_property CONSTANT)
```

Comment:
I think this service has no place here 
I think it would be better to have this solution:
```
ToursProvider.qml
{
 
         ToursProviderModel {   // or ToursProviderQml, ToursProviderView

         }
}


class ToursProviderModel 
{
       Q_OBJECT

       Inject<tours::IToursProvider> toursProvider;
...
}

```